www/includes/easyparliament/templates/html/api/update.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+$stripe = $subscription->stripe;
+$stripeToken = get_http_var('stripeToken');
+$charitable_tick = $stripe ? $stripe->discount : '';
+$charitable = $stripe ? $stripe->metadata['charitable'] : '';
+$charity_number = $stripe ? $stripe->metadata['charity_number'] : '';
+$description = $stripe ? $stripe->metadata['description'] : '';
+
+function rdio($name, $value, $text, $id, $required=false, $checked=false) {
+?>
+    <li><label for="<?= $id ?>"><input type="radio" name="<?= $name ?>" value="<?= $value ?>" id="<?= $id ?>"
+    <?= $required ? 'required' : '' ?>
+    <?= $value == $checked ? 'checked' : '' ?>>
+    <?= $text ?></label></li>
+<?php
+}
+
+?>
+
+<h2><?= $stripe ? "Change plan" : "Subscribe to a plan" ?></h2>
+
+<?php if ($errors && $stripeToken) { ?>
+  <div class="account-form__errors">
+    We have safely stored your payment information, and you have not yet been charged.
+    Please correct the issues below, you will not need to reenter your card details.
+  </div>
+<?php } ?>
+
+<?php
+if ($errors) {
+    print "<ul>";
+    foreach ($errors as $error) {
+        print "<li>$error</li>";
+    }
+    print "</ul>";
+}
+?>
+
+<?php if ($stripe && $stripe->cancel_at_period_end) { ?>
+<p>Your plan is curently set to expire on <?= date('d/m/Y', $stripe->current_period_end) ?>.
+If you update your plan below, it will be reactivated.
+</p>
+<?php } ?>
+
+<form id="signup_form" method="post" action="/api/update-plan" autocapitalize="off">
+
+
+<div class="row">
+    <label for="id_plan_0">Please choose a plan:</label>
+    <ul id="id_plan">
```

Comment:
We should say this is ex VAT as I was confused that it said 20 here and then 24 later on in the process.

Reply:
Ah, that's my mistake setting it up. It will be inc VAT and should be the same in both places, but I mismatched the pricing in test Stripe.